docs/admin/install/docker.rst

```diff
@@ -1824,6 +1832,19 @@ Container settings
         :doc:`Celery worker options <celery:reference/celery.bin.worker>`,
         :ref:`celery`
 
+.. envvar:: CELERY_SINGLE_PROCESS
```

Comment:
Needs version annotation once it is clear where https://github.com/WeblateOrg/docker/pull/2568 lands:

```suggestion
.. envvar:: CELERY_SINGLE_PROCESS

   .. versionadded:: 5.7.1
```